### DISCUSSION TITLE
Transient solve; steady-state detection on postprocessor output

### DISCUSSION THREAD

**Discussion by jvwilliams23** [idaholab/moose#27404] [Q&A General] [answered]
Hi,

I am wondering if it is possible to specify a specific quantity to base the steady-state detection on when `steady_state_detection = true`? I see there is a `check_aux` option, but this is a bool and so not useful for pointing to a specific aux variable.
I want to stop the simulation when the change in the **maximum** temperature is steady (not interested in other dynamics).

So I am looking for something similar to this:

```
[Postprocessors]
  [max-temp]
    type = NodalExtremeValue
    variable = temp
  []
[]

[Executioner]
  type = Transient
  steady_state_detection = true
  steady_state_monitor = max-temp # Does something like this exist?
[]
```

I am looking to do some UQ on a big transient simulation, so I am trying to find performance savings to minimise excess computation.

Best,
Josh

**Comment by GiudGiud** [comment, marked as answer]
Hello

You cannot do this with the executioner parameters currently. You will soon be able to do thanks to development of a new Convergence system that will have many options.

For now,use a terminator user object to finish the simulation and a postprocessor to measure the criterion (change in the variable)

Guillaume

**Reply by jvwilliams23** [reply]
Thanks, I just checked Terminator. It looks like exactly what I need!
